src/locales/en.ts

```diff
@@ -139,10 +139,10 @@ export default {
 	},
 	documents: {
 		title: "Documents",
-		subtitle: "Discover the official documents of Capital Technology Network.",
+		subtitle: "Discover the official documents of The Capital Technology Network.",
 		constitution: "Constitution",
 		constitution_desc:
-			"The official constitution of Capital Technology Network, outlining the organization's structure and purpose.",
+			"The official constitution of The Capital Technology Network, outlining the organization's structure and purpose.",
```

Comment:
"The" should be lowercase, it's not part of the proper noun itself, it's just a determiner